File: csgo-leagues-ios/csgo-leagues-ios/Data/DTOs/MatchDTO.swift
struct MatchDTO: Codable {
    let id: Int
    let name: String
    let status: String
    let beginAt: String?      // decoded from "begin_at", ISO8601
    let opponents: [OpponentContainerDTO]
    let league: LeagueDTO
    let serie: SerieDTO
}

File: csgo-leagues-android/app/src/main/java/com/voidx/csgo/data/dto/MatchDTO.kt
package com.voidx.csgo.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MatchDTO(
    val id: Int,
    val name: String = "",
    val status: String,
    @SerialName("begin_at") val beginAt: String? = null,
    val opponents: List<OpponentContainerDTO> = emptyList(),
    val league: LeagueDTO,
    val serie: SerieDTO,
)
